README.rst

```diff
@@ -1,15 +1,119 @@
+.. image:: https://github.com/crate/crate-operator/workflows/Build%20and%20publish%20Docker%20Image/badge.svg
+   :alt: Docker Build
+   :target: https://github.com/crate/crate-operator
 
-Development
-===========
+.. image:: https://img.shields.io/badge/docs-latest-brightgreen.svg
+   :alt: Documentation
+   :target: https://crate-operator.readthedocs.io/en/latest/
 
-The ``crate-operator`` package requires **Python 3.8**. It is build ontop of
-the `Kopf: Kubernetes Operators Framework`_.
+.. image:: https://img.shields.io/badge/container-docker-green.svg
+   :alt: Docker Hub
+   :target: https://hub.docker.com/crate/crate-operator/
+
+
+=============================
+⚙️ CrateDB Kubernetes Operator
+=============================
+
+The **CrateDB Kubernetes Operator** provides convenient way to run `CrateDB`_
+clusters inside `Kubernetes`_. It is built on top of the `Kopf: Kubernetes
+Operators Framework`_.
+
+🗒️ Contents
+==========
+
+- `🤹 Usage`_
+- `🎉 Features`_
+- `💽 Installation`_
+- `💻 Development`_
+
+🤹 Usage
+=======
+
+A minimal custom resource for a 3 node CrateDB cluster may look like this:
+
+``dev-cluster.yaml``:
+
+.. code-block:: yaml
+
+   apiVersion: cloud.crate.io/v1
+   kind: CrateDB
+   metadata:
+     name: my-cluster
+     namespace: dev
+   spec:
+     cluster:
+       imageRegistry: crate
+       name: crate-dev
+       version: 4.3.1
+     nodes:
+       data:
+       - name: default
+         replicas: 3
+         resources:
+           cpus: 4
+           disk:
+             count: 1
+             size: 128GiB
+             storageClass: default
+           heapRatio: 0.5
+           memory: 4Gi
+
+.. code-block:: console
+
+   $ kubectl --namespace dev create -f dev-cluster.yaml
+   ...
+
+   $ kubectl --namespace dev get cratedbs
+   NAMESPACE   NAME         AGE
+   dev         my-cluster   36s
+
+🎉 Features
+==========
+
+- "all equal nodes" cluster setup
+- "master + data nodes" cluster setup
+- safe scaling of clusters
+- safe rolling version upgrades for clusters
+- SSL for HTTP and PG connections via Let's Encrypt certificate
+- custom node settings
+- custom cluster settings
+- custom storage classes
+- region/zone awareness for AWS and Azure
+
+💽 Installation
+==============
+
+To be able to deploy the custom resource ``CrateDB`` to a Kubernetes cluster,
+the API needs to be extended with a `Custom Resource Definition` (CRD). The CRD
+for ``CrateDB`` can be found in the ``deploy/`` folder and can be applied
+(assuming sufficient privileges).
+
+.. code-block:: console
+
+   $ kubectl apply -f deploy/crd.yaml
+   customresourcedefinition.apiextensions.k8s.io/cratedbs.cloud.crate.io created
+
+Once the CRD is installed, the operator itself can be deployed using a
+``Deployment``.
+
+.. code-block:: console
+
+   $ kubectl create -f deploy/rbac.yaml
+   ...
+   $ kubectl create -f deploy/deployment.yaml
+   ...
+
+Please refer to the `configuration documentation`_ for further details.
+
+💻 Development
```

Comment:
I think we could move the whole development section into https://github.com/crate/crate-operator/blob/master/docs/source/development.rst instead.

Reply:
Yeah, makes sense